### DISCUSSION TITLE
Set CLR object prototype

### DISCUSSION THREAD

**Discussion by mainlyer** [sebastienros/jint#1509] [Q&A] [answered]
Hi, is there a way to keep inheritance from CLR objects in order to make instaceof work in javascript?

Thank you in advance.

**Comment by lahma** [comment, marked as answer]
We haven't explored that would be happy to accept a PR enabling that (if missing) with accompanied test cases to cover the functionality.

**Comment by mainlyer** [comment]
Thanks for the help @lahma. Take a look to the PR #1511.

Regards.
